rewards/boost/calc_boost.py

```diff
@@ -67,6 +69,10 @@ def badger_boost(current_block: int, chain: str) -> Dict[str, Any]:
 
     for user, ratio in stake_ratios.items():
         boost_info[user]["stakeRatio"] = ratio
+        
+    for user, nft_balances in nfts.items():
+        if user in all_addresses:
```

Comment:
Same here, and a few other of the `for` loops.  `allocate_nft_balances_to_users(all_addresses, nfts)` is clear in the underlying logic of what's happening, and allows for unit testing.  `assign_stake_ratio_to_users`, `allocate_sett_balances_to_users`, etc.

Reply:
I'm also OK with logging this and the above refactoring into an issue and adding as tech debt cleanup + testing improvement as this is a non-blocker.